platform/android/SCsub

```diff
@@ -83,7 +83,8 @@ if lib_arch_dir != "":
     stl_lib_path = (
         str(env["ANDROID_NDK_ROOT"]) + "/sources/cxx-stl/llvm-libc++/libs/" + lib_arch_dir + "/libc++_shared.so"
     )
-    env_android.Command(out_dir + "/libc++_shared.so", stl_lib_path, Copy("$TARGET", "$SOURCE"))
+    # FIXME: `sources/llvm-libc++` removed in NDK 26 and later. Commenting out just so the C++20 builds complete.
+    # env_android.Command(out_dir + "/libc++_shared.so", stl_lib_path, Copy("$TARGET", "$SOURCE"))
```

Comment:
Highlighting this because I'm gonna need someone from the @godotengine/android team to clarify matters. C++20 support is added to Android starting with NDK26, which I'm now targeting for this PR, but this shared object is no longer here starting with this version. I *believe* this is deliberate, as the NDK26 changelog[^1] says this:
> libc++ has been updated. The NDK's libc++ now comes directly from our LLVM toolchain, so every future LLVM update is also a libc++ update. 

I have no idea what the full ramifications of this are, beyond it presumably being the reason `sources/cxx-stl/llvm-libc++` and `sources/cxx-stl/llvm-libc++abi` were removed. There's still shared libraries available in NDK26 in the prebuilt section — `toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/<arch_alias>-linux-android` — but that section was already available in NDK23 so maybe it wasn't being used for a reason?

[^1]:https://github.com/android/ndk/wiki/Changelog-r26